jar-infer/jar-infer-lib/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     }
     testCompile project(":jar-infer:test-java-lib-jarinfer")
     testCompile project(path: ":jar-infer:test-android-lib-jarinfer", configuration: "default")
+    testCompile files("${System.properties['java.home']}/../lib/tools.jar") // is there a better way?
```

Comment:
@manu @navahgar Any thoughts? Is there a Maven Central friendly version of this jar? I was not able to find one...

Reply:
Not that I know of. :(